app/experimenter/experiments/tests/test_serializers.py

```diff
@@ -388,7 +388,7 @@ def test_serializer_outputs_expected_schema(self):
 class TestExperimentRecipePrefArgumentsSerializer(TestCase):
 
     def test_serializer_outputs_expected_schema(self):
-        experiment = ExperimentFactory.create_with_status(Experiment.STATUS_SHIP)
+        experiment = ExperimentFactory(pref_type=Experiment.PREF_TYPE_JSON_STR)
         serializer = ExperimentRecipePrefArgumentsSerializer(experiment)
```

Comment:
maybe keep the original and a modified test that tests for json string? we're boxing ourselves in to only test for json string